metals/src/main/scala/scala/meta/internal/metals/WorkspaceSearchVisitor.scala

```diff
@@ -28,7 +27,7 @@ class WorkspaceSearchVisitor(
     workspace: AbsolutePath,
     query: WorkspaceSymbolQuery,
     token: CancelChecker,
-    index: OnDemandSymbolIndex,
+    index: scala.meta.internal.mtags.GlobalSymbolIndex,
```

Comment:
It would be more consistent to keep it as `index: GlobalSymbolIndex` and do proper import (also similar in def below)

Reply:
Thanks, should be fixed now. Seems I had underlooked that commit.